source/module_io/input.h

```diff
@@ -271,7 +271,10 @@ class Input
     std::vector<int> out_band; // band calculation pengfei 2014-10-13
     bool out_proj_band; // projected band structure calculation jiyy add 2022-05-11
     std::vector<int> out_mat_hs; // output H matrix and S matrix in local basis.
-    bool out_mat_xc; // output exchange-correlation matrix in KS-orbital representation.
+    bool out_mat_xc; 
+    bool out_hr_npz;// output exchange-correlation matrix in KS-orbital representation.
+    bool out_dm_npz;
+    bool dm_to_rho;
```

Comment:
It is quite hard to understand what `dm_to_rho` control, could you add some docs about it?

Reply:
this feature is for internal use only; I am not inclined to add a documentation such as to give the impression that it is also intended for external users. Let me know if we want to discuss on this matter